aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -184,6 +184,37 @@ void div_mode_template(const Tensor& self, const Tensor& other,
   binaryOpTensor(self, other, Scalar(1.0), output, op_name + "_out_mps:" + (rounding_mode.has_value() ? c10::str(*rounding_mode) : ""), div_mode_op_block);
 }
 
+void fmod_template(const Tensor& self, const Tensor& other,
+                   const Tensor& output, const string op_name)
+{
+  BinaryOpBlock fmod_op_block = ^BinaryOpFn(cachedGraph, primaryCastTensor, secondaryCastTensor) {
+    MPSGraph* mpsGraph = cachedGraph->graph();
+    MPSGraphTensor* absPrimaryTensor = [mpsGraph absoluteWithTensor:primaryCastTensor
+                                                               name:nil];
+    MPSGraphTensor* absSecondaryTensor = [mpsGraph absoluteWithTensor:secondaryCastTensor
+                                                                 name:nil];
+    MPSGraphTensor* modTensor = [mpsGraph floorModuloWithPrimaryTensor:absPrimaryTensor
```

Comment:
This doesn't have support for i64 as mentioned below. Can you raise a TORCH_CHECK error for it?